socorro/external/boto/crashstorage.py

```diff
@@ -293,12 +304,37 @@ def save_raw_and_processed(
         processed_crash,
         crash_id
     ):
-        schema = PROCESSED_CRASH_JSON_SCHEMA
-        crash = json_schema_reducer.make_reduced_dict(schema, processed_crash)
+        all_fields = SuperSearchFields(config=self.config).get()
```

Comment:
It's not that simple. If processors were to cache that value, we would have to restart them every time we change any value in the SuperSearchFields list. I agree that querying this for every single crash is too much though. 

I think this is the same problem that we had with having the signature lists in the database. I don't remember us finding a good solution (and I do not really consider having to kill the processor threads to be a good solution).